README.md

```diff
@@ -1,2 +1,35 @@
 ## This repository is for documentation of the [GeoBlacklight](https://geoblacklight.org) application.
 
+### Local Build
+
+1. Clone this repo (or your own fork of it)
+
+    ```
+    git clone https://github.com/geoblacklight/docs && cd docs
+    ```
+
+2. Create and activate a Python virtual environment with the tool of your choice ([venv](https://docs.python.org/3/library/venv.html), [conda](https://docs.conda.io/projects/conda), etc.). The following example uses `venv`.
+
+    ```
+    python3 -m venv env && source env/bin/activate
```

Comment:
I've never heard of venv before (Rubyist). Is using venv common and/or better than just stock `pip install mkdocs` and `mkdocs serve`?

Reply:
Virtual environments are definitely a best practice in Python--you want to avoid pip installing packages into your stock Python installation because certain operating system tools may rely on specific versions of specific packages, and you could inadvertently break those dependencies. A virtual environment provides a "sandbox" where you can install whatever you want without affecting the system. `venv` comes with Python 3.5+, so it's what I like to use, but there are a lot of other (too many) other ways to create and manage virtual environments as well.

Reply:
Thanks for the background here!